draft-vvv-webtransport-http3.md

```diff
@@ -413,23 +397,27 @@ Description:
 
 Reference:
 
-: This document
+: This document and {{?I-D.kinnear-webtransport-http2}}
+
+## HTTP/3 SETTINGS Parameter Registration
+
+The following entry is added to the "HTTP/3 Settings" registry established by
+[HTTP3]:
 
-## QUIC Transport Parameter Registration
+The `SETTINGS_ENABLE_WEBTRANSPORT` parameter indicates that the specified
+HTTP/3 connection is WebTransport-capable.
 
-The following entry is added to the "QUIC Transport Parameter Registry" registry
-established by [QUIC-TRANSPORT]:
+Setting Name:
 
-The `http3_transport_support` parameter indicates that the specified HTTP/3
-connection is WebTransport-capable.
+: ENABLE_WEBTRANSPORT
 
 Value:
 
-: 0x????
+: 0x???? (currently H2 draft uses 0xFB)
```

Comment:
Can we pick a value in the 4-byte encoding range to allow folks to test interop?